couchdb.py

```diff
@@ -106,3 +106,16 @@ def dbcontext(dbname=MATCHES_DBNAME):
     ) as couch_client:
         yield couch_client[dbname]
 
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--display-stats", action="store_true")
+    args = parser.parse_args()
+
+    if args.display_stats:
+        matches_db = get_matches_db()
+        all_docs = matches_db.all_docs()
+        print(f"Num fully parsed matches: {all_docs['total_rows']}")
+        print(
+            f"Last match start_time {get_last_match_by_start_time(matches_db)['start_time']}"
+        )
```

Comment:
while you're in here, might as well wrap this time in `datetime.datetime.fromtimestamp()` so we can read it.